libs/components/lookup/src/lib/modules/search/search.component.ts

```diff
@@ -320,10 +346,25 @@ export class SkySearchComponent implements OnDestroy, OnInit, OnChanges {
     this.#changeRef.markForCheck();
   }
 
-  #searchShouldCollapse() {
+  #searchShouldCollapse(): boolean {
     return (
       (this.isCollapsible || this.isCollapsible === undefined) &&
       this.isFullWidth !== true
     );
   }
+
+  #setupSearchChangedEvent(): void {
+    this.#searchUpdatedUnsubscribe.next();
```

Comment:
Without calling `this.#searchUpdatedUnsubscribe.complete()` here, the previous subscription would never be unsubscribed. I'd prefer tracking the `Subscription` itself rather than a separate `Subject` since it's only used for this one `Subject`.

```
  this.#searchUpdatedSub?.unsubscribe();

 this.#searchUpdatedSub = this.#searchUpdated...subscribe(...)
```